playground/OracleEndToEnd/OracleEndToEnd.AppHost/aspire-manifest.json

```diff
@@ -0,0 +1,294 @@
+{
+  "$schema": "https://json.schemastore.org/aspire-8.0.json",
+  "resources": {
+    "pg1": {
```

Comment:
This doesn't look correct. Did you just copy-paste this?

Reply:
Yes, and you're right definitely not accurate. Do I need to generate one or can I just delete the file?

Reply:
Regenerate it.

Reply:
Done